### DISCUSSION TITLE
Using Icons from SVG Factory

### DISCUSSION THREAD

**Discussion by kyle-power** [Appsilon/shiny.fluent#20] [Q&A] [answered]
So I am trying to include SVG Factory icons in my `DocumentCard()`, specifically `DocumentCardPreview()`.
According to the FluentUI documentation, this can be achieved using the `previewIconProps` argument, however in the FluentUI documentation, the following is called at the beginning of the script...
```
import * as ReactIcons from '@fluentui/react-icons-mdl2';
```
which is later referenced in the `DocumentCardPreview()` - see here: https://developer.microsoft.com/en-us/fluentui#/controls/web/icon

How can I `import` the icons for use in my `{shiny.fluent}` app?

**Comment by kamilzyla** [comment]
Hi @kyle-power! Thanks for the question, I'll take a look next week :slightly_smiling_face:

**Comment by kamilzyla** [comment, marked as answer]
Hi @kyle-power! The `@fluentui/react-icons-mdl2` is a separate JavaScript library and it is not included in `shiny.fluent`. There is no simple way to add it using just R - you'd need to use [NPM](https://en.wikipedia.org/wiki/Npm_(software)) and some bundler (e.g. [webpack](https://en.wikipedia.org/wiki/Webpack)) to create a JS file ready for inclusion in your app. Alternatively, you could find the library on some CDN (however, my quick search didn't yield any results).

With that said, we might be able to add this icon library to shiny.fluent - I'll take it as a feature request :slightly_smiling_face:
